src/helperFunctions/docker.py

```diff
@@ -1,48 +1,55 @@
 import logging
 from contextlib import suppress
-from typing import Optional, Tuple
 
 import docker
-from docker.errors import DockerException
-from docker.types import Mount
-from requests.exceptions import ReadTimeout
-
-
-def run_docker_container(  # pylint: disable=too-many-arguments
-    image: str, timeout: int = 300, command: Optional[str] = None, reraise: bool = False, privileged: bool = False,
-    mount: Optional[Tuple[str, str]] = None, label: str = 'Docker', include_stderr: bool = True
-) -> str:
-    '''
-    Run a docker container and get its output.
-
-    :param image: the name of the docker image
-    :param timeout: a timeout after which the execution is canceled
-    :param command: the command to run in the container (optional)
-    :param reraise: re-raise exceptions if they occur (timeout and docker exceptions)
-    :param privileged: Run container with elevated privileges.
-    :param mount: specifies a directory that gets mounted into the container;
-                  structure: `(path_inside_container, source_path)`
-    :param label: label used for logging output
-    :param include_stderr: include stderr of the container in the output
-    :return: the output of the docker container
-    '''
-    container = None
+from docker.errors import APIError, DockerException, ImageNotFound
+
+client = docker.client.from_env()
+
+
+def run_docker_container(image: str, logging_label: str = 'Docker', timeout=300,  stderr=True, **kwargs):
```

Comment:
Just one thing to consider: Is 5 minutes really a good default timeout?
It seems long.